assemblies/assembly-planner-configuration.adoc

```diff
@@ -50,20 +50,20 @@ OptaPlanner makes it relatively easy to switch optimization algorithms by changi
 .. Define the score function.
 .. Optional: Configure the optimization algorithm.
 +
-The following example is a solver XML file for the NQueens problem:
+The following example is a solver XML file for the time table problem:
 +
 [source,xml,options="nowrap"]
 ----
 <?xml version="1.0" encoding="UTF-8"?>
 <solver xmlns="https://www.optaplanner.org/xsd/solver" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="https://www.optaplanner.org/xsd/solver https://www.optaplanner.org/xsd/solver/solver.xsd">
   <!-- Define the model -->
-  <solutionClass>org.optaplanner.examples.nqueens.domain.NQueens</solutionClass>
-  <entityClass>org.optaplanner.examples.nqueens.domain.Queen</entityClass>
+  <solutionClass>org.optaplanner.examples.timetable.domain.timetable</solutionClass>
```

Comment:
As far as I know there is no timetable example in optaplanner package so in case you want to describe here configuration of Solver you woud need to use "Using the Java API to configure the OptaPlanner solver" chapter. 
Look here, it has only Java Api configuration
http://github.com/dupliaka/optaplanner-quickstarts/blob/3201223279c687215764c88fd962930b41329ed0/hello-world/src/main/java/org/acme/schooltimetabling/TimeTableApp.java#L46-L46

So the part about 'Using the Java API to configure the OptaPlanner solver' should be changed as well since the current creates the config from xml.